### PR TITLE
Scripts: Spell-check for windows

### PR DESCRIPTION
**Pull request by marqdevx**
## What This PR Changes
- Add spell-check for windows

### CODE REVIEW

File: scripts/spell-check.cmd [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+codespell --version
+IF %ERRORLEVEL% NEQ 0 (
+    ECHO Installing codespell
+    pip install codespell || pip3 install codespell
+) else (
+    ECHO Codespell already installed
+)
+cls
+codespell -I scripts/resources/spell-check-ignore-list.txt --skip="*.svg,*.dxf" content
+ECHO Spell-check finished
```
Comment (sebromero) [inline scripts/spell-check.cmd:20]:
```suggestion
```
I'd keep the output the same as with the Unix version. VSCode should indicate when the task is done, so imo no extra message needed.

**Review by sebromero — CHANGES_REQUESTED** [review]
Almost ready to go :-)

File: scripts/spell-check.cmd (hunk 1 above)
Comment (marqdevx) [reply scripts/spell-check.cmd:20]: Oops, didnt look at the unix version echo :S
Removing it

**Review by sebromero — APPROVED** [review]
LGTM

**Pull request merged**